spec/index.spec.ts

```diff
@@ -43,3 +44,28 @@ import './providers/pubsub.spec';
 import './providers/remoteConfig.spec';
 import './providers/storage.spec';
 import './providers/crashlytics.spec';
+
```

Comment:
I think this file is for consolidating all the test files so that they run together when you do `npm run test`. Can we have a separate file for testing setup? That way we don't need all those imports at the top which at first confused me.

Reply:
moved to `setup.spec`